MerlinAU.sh

```diff
@@ -1260,8 +1260,16 @@ _CreateEMailContent_()
              printf "\nPlease check <b>backupmon.sh</b> configuration and retry F/W Update from current version:\n<b>${fwInstalledVersion}</b>\n"
            } > "$tempEMailBodyMsg"
            ;;
+       FAILED_FW_UNZIP_STATUS)
```

Comment:
Smart, I like it, give the user the reason it failed if it's run by cron they can clearly see the step it failed at and why it would appear to them as though it "didn't appear to update/flash"

Reply:
> Smart, I like it, give the user the reason it failed if it's run by cron they can clearly see the step it failed at and why it would appear to them as though it "didn't appear to update/flash"
>
Exactly. That was the goal: to provide the reason why the update didn't happen at that point.